### DISCUSSION TITLE
Ability to change Field Instance to different Field

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/cms#15033] [Ideas]
Hello,
sometimes it happens to me, that I use configured Field in multiple Entry Types and later find out I need to change the Field configuration for some of the Entry Types.

Let me give you simple example.
I create Plain Text Field configured to allow line breaks. I use this field in multiple Entry Types. Sometimes for titles, sometimes for description.
Later I realize that titles shouldn't allow line breaks. But I already use it for descriptions where I need line breaks.
I cannot change the Field configuration and disable line breaks but I also want to disable line breaks for some cases of usage.

Ideally I would create new Plain Text Field with disabled line breaks and then go to affected Entry Types and change the Field Instance so it points to different Field. This way I can change only some Field Instances while keep others.

Currently I do it manually in Project Config where I find the Field Instance I need and change fieldUid so it points to another Field.
I just thought maybe it could be possible to do in administration.

Thanks.

**Comment by thupsi** [comment]
FWIW you can also copy the values of a specific field to a new field via CLI. I agree though that it would be nice to have this ability in the CP. See also https://github.com/craftcms/cms/discussions/14566
